### PR TITLE
Display data producer fragment

### PR DESCRIPTION
**Pull request by nguyenvanhieu97**
https://edu-redmine.framgia.vn/issues/5992

### CODE REVIEW

**Comment by nguyenvanhieu97** [comment]
screenshot 24 jan 2019 14_03_42

**Comment by nguyenvanhieu97** [comment]
ready

File: app/src/main/res/layout/fragment_producer.xml [hunk 1]
```diff
@@ -11,17 +17,23 @@
             android:id="@+id/text_producer_result"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/dp_10"
             android:layout_marginStart="@dimen/dp_5"
+            android:text="@{`Result: ` + String.valueOf(movieVM.movieObservable.companies.size())}"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_producer.xml]: xử lý lại cho anh.

Comment (nguyenvanhieu97) [reply app/src/main/res/layout/fragment_producer.xml]: vâng

**Comment by nguyenvanhieu97** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**